src/sign.c

```diff
@@ -22,88 +22,87 @@ typedef struct sign sign_T;
-# ifdef FEAT_SIGN_ICONS
-    void	*sn_image;	// icon image
-# endif
-    char_u	*sn_text;	// text used instead of pixmap
-    int		sn_line_hl;	// highlight ID for line
-    int		sn_text_hl;	// highlight ID for text
-    int		sn_cul_hl;	// highlight ID for text on current line when 'cursorline' is set
-    int		sn_num_hl;	// highlight ID for line number
-    int		sn_priority;	// default priority of this sign, -1 means SIGN_DEF_PRIO
+    sign_T *sn_next; // next sign in list
+    int sn_typenr;   // type number of sign
+    char_u *sn_name; // name of sign
+    char_u *sn_icon; // name of pixmap
+#ifdef FEAT_SIGN_ICONS
+    void *sn_image;  // icon image
+#endif
+    char_u *sn_text; // text used instead of pixmap
+    int sn_line_hl;  // highlight ID for line
+    int sn_text_hl;  // highlight ID for text
+    int sn_cul_hl; // highlight ID for text on current line when 'cursorline' is set
+    int sn_num_hl;   // highlight ID for line number
+    int sn_priority; // default priority of this sign, -1 means SIGN_DEF_PRIO
 };
 
-static sign_T	*first_sign = NULL;
-static int	next_sign_typenr = 1;
+static sign_T *first_sign = NULL;
+static int next_sign_typenr = 1;
 
 static void sign_list_defined(sign_T *sp);
 static void sign_undefine(sign_T *sp, sign_T *sp_prev);
 
-static char *cmds[] = {
-			"define",
-# define SIGNCMD_DEFINE	0
-			"undefine",
-# define SIGNCMD_UNDEFINE 1
-			"list",
-# define SIGNCMD_LIST	2
-			"place",
-# define SIGNCMD_PLACE	3
-			"unplace",
-# define SIGNCMD_UNPLACE 4
-			"jump",
-# define SIGNCMD_JUMP	5
-			NULL
-# define SIGNCMD_LAST	6
+static char *cmds[] = { "define",
+#define SIGNCMD_DEFINE 0
+                        "undefine",
+#define SIGNCMD_UNDEFINE 1
+                        "list",
+#define SIGNCMD_LIST 2
+                        "place",
+#define SIGNCMD_PLACE 3
+                        "unplace",
+#define SIGNCMD_UNPLACE 4
+                        "jump",
+#define SIGNCMD_JUMP 5
+                        NULL
+#define SIGNCMD_LAST 6
 };
 
-#define FOR_ALL_SIGNS(sp)	\
+#define FOR_ALL_SIGNS(sp) \
     for ((sp) = first_sign; (sp) != NULL; (sp) = (sp)->sn_next)
 
-static hashtab_T	sg_table;	// sign group (signgroup_T) hashtable
-static int		next_sign_id = 1; // next sign id in the global group
+static hashtab_T sg_table;   // sign group (signgroup_T) hashtable
+static int next_sign_id = 1; // next sign id in the global group
 
-/*
+                             /*
  * Initialize data needed for managing signs
  */
-    void
+void
 init_signs(void)
 {
-    hash_init(&sg_table);		// sign group hash table
+    hash_init(&sg_table); // sign group hash table
 }
 
 /*
  * A new sign in group 'groupname' is added. If the group is not present,
  * create it. Otherwise reference the group.
  */
-    static signgroup_T *
+static signgroup_T *
 sign_group_ref(char_u *groupname)
 {
-    hash_T		hash;
-    hashitem_T		*hi;
-    signgroup_T		*group;
+    hash_T hash;
+    hashitem_T *hi;
+    signgroup_T *group;
```

Comment:
The Vim coding convention for local variable declarations is to align the variable names (using 'softtabstop').

Reply:
Also acknoledge in at the end of the description

> * Reliable alignment of varibles names in structs and functions.
>   * There is an option for clang format to align variable names but is really clunky and doesn't work well with the preprocessor

Again this minor change. A formatter is a far better tool to have that alligned variable names.

Reply:
Ok for some strange reasons it doesn't compile anymore if I format all the project.